CMakeLists.txt

```diff
@@ -199,14 +199,6 @@ if (UNIX)
                 int main() { XdpOutputType out = XDP_OUTPUT_NONE; }
             " HAVE_LIBPORTAL_OUTPUT_NONE)
             cmake_pop_check_state()
-
-            if(INPUTLEAP_DEPLOY_FLATPAK_SCRIPT)
```

Comment:
You should remove the option for this. (line 27 of this file)

Reply:
done